http-server-netty/src/test/groovy/io/micronaut/http/server/netty/handler/accesslog/AccessLogSpec.groovy

```diff
@@ -358,15 +359,15 @@ class AccessLogSpec extends Specification {
             responses.size() == 3
         }
         responses[0].content().toString(StandardCharsets.UTF_8) == 'simple'
-        responses[1].content().toString(StandardCharsets.UTF_8) == 'open'
-        responses[2].content().toString(StandardCharsets.UTF_8) == 'finish'
```

Comment:
Is it correct that `finish` now comes before `open`?  Http2 confuses me 😢

Reply:
same question for me

Reply:
the controller for 'open' waits for a future that is completed by the controller for 'finish'. so basically the call to 'finish' immediately returns both the 'open' and the 'finish' responses. the order of the responses doesn't really matter, they should arrive at the same time, but iirc it's flipped here because the multiplexer delays the flush of the 'open' response until the 'finish' response is also written